hw02-arraylist/src/main/java/com/remifo/DIYArrayList.java

```diff
@@ -0,0 +1,280 @@
+package com.remifo;
+
+import java.util.*;
+
+public class DIYArrayList<T> implements List<T> {
+    private Object[] array = new Object[0];
```

Comment:
Логично сделать сразу не нулевой массив.
```suggestion
    private Object[] array = new Object[DEFAULT_CAPACITY];
```
И тогда `DEFAULT_CAPACITY` должен быть перед этой строчкой.

Reply:
Да, действительно, логично)